src/components/RadioGroup/RadioGroupTypes.tsx

```diff
@@ -0,0 +1,6 @@
+export interface Option {
```

Comment:
It would be reasonable to export this from RadioGroup.tsx instead of creating a RadioGroupTypes.tsx file.

Reply:
After experimenting with this in TextInput, I found that it really cluttered up the head of the file. I'm opting to keep the separate type files, to maintain simplicity in the actual components, and to make it obvious that components have types which might be used in other cases.

If this is ever the case, the type definition will be moved to /lib/types.tsx.